README.md

```diff
@@ -221,7 +221,7 @@ A collection of awesome browser-side [JavaScript](https://developer.mozilla.org/
     * [derby-awesome](https://github.com/russll/awesome-derby) - A collection of awesome derby components
 * [way.js](https://github.com/gwendall/way.js) - Simple, lightweight, persistent two-way databinding.
 * [mithril.js](https://github.com/lhorie/mithril.js) - Mithril is a client-side MVC framework (Light-weight, Robust, Fast).
-* [jsblocks](https://github.com/astoilkov/jsblocks) - jsblocks is better MV-ish framework.
+* [jsblocks](https://github.com/astoilkov/jsblocks) - jsblocks is a better MV-ish framework.
```

Comment:
Their own description is 
> Better MV-ish Framework

but the project was archived. hard to tell if it's still _better_ at anything :P